tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,48 +87,69 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val wskpropsBackup = TestUtils.backupWskProps()
+        wsk.cli(Seq("property", "unset", "--auth", "--apihost", "--apiversion", "--namespace"))
+        var stdout = wsk.cli(Seq("property", "get", "--auth")).stdout
+        stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--apihost")).stdout
+        stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--namespace")).stdout
+        stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+        TestUtils.restoreWskProps(wskpropsBackup)
+        wskpropsBackup.delete()
+    }
+
     it should "show api build version" in {
-        val stdout = wsk.cli(wskprops.overrides ++ Seq("property", "get", "--apibuild")).stdout
+        val wskpropsBackup = TestUtils.backupWskProps()
+        wsk.cli(wskprops.overrides ++ Seq("property", "set"))
```

Comment:
this should have no effect: `wsk property set` is a no op. This may be a point of divergence between the semantics of arguments and their position in python vs go. In python `wsk --auth xxx property set` is not the same as `wsk property set --auth xxx`. In go cli, from this test, I'm guess that they are the same.